docs/source/nlp/tutorials/quickstarts/plot_text_classification.py

```diff
@@ -23,206 +31,267 @@
+# run them stand-alone.
+#
+# Embeddings Drift
+# ----------------
+#
+# In order to run the embedding drift check you must have text embeddings
+# loaded to both datasets. You can read more about using embeddings in deepchecks NLP in our
+# :ref:`Embeddings Guide <nlp__embeddings_guide>`.
+#
+# In this example, we have the embeddings already
+# pre-calculated:
 
-# Calculate properties
-train.calculate_default_properties()
-test.calculate_default_properties()
+from deepchecks.nlp.datasets.classification.tweet_emotion import load_embeddings
 
-# Run the check
-from deepchecks.nlp.checks import PropertySegmentsPerformance
+train_embeddings, test_embeddings = load_embeddings()
 
-check = PropertySegmentsPerformance(segment_minimum_size_ratio=0.07)
-result = check.run(test, predictions=list(test_preds), probabilities=test_probas)
-result.show()
+train.set_embeddings(train_embeddings)
+test.set_embeddings(test_embeddings)
+
+# %%
+# You can also calculate the embeddings using deepchecks, either using an
+# open-source sentence-transformer or using Open AI’s embedding API.
 
-#%%
-# As we can see, the check found some segments that have lower performance compared to the rest of the dataset. It seems
-# that the model has a harder time predicting the emotions in the "neutral-positive" sentiment range (in our case,
-# between around 0 and 0.45).
+# train.calculate_default_embeddings()
+# test.calculate_default_embeddings()
+
+# %%
 #
-# Data Integrity Checks
-# ---------------------
+
+from deepchecks.nlp.checks import TextEmbeddingsDrift
+
+check = TextEmbeddingsDrift()
+res = check.run(train, test)
+res.show()
+
+# %%
+# Here we can see some distinct segments that distinctly contain more
+# samples from train or more sample for test. For example, if we look at
+# the greenish cluster in the middle we see it’s full of inspirational
+# quotes and saying, and belongs mostly to the test dataset. That is the
+# source of the drastic increase in optimistic labels!
 #
-# These previous checks were all about the model's performance. Now we'll run a check that attempts to find instances
-# of shortcut learning - cases in which the label can be predicted by simple aspects of the data, which
-# in many cases can be an indication that the model has used some information that won't generalize to the real world.
+# There are of course also other note-worthy segments, such as the greenish cluster on the right that contains tweets
+# about a terror attack in Bangladesh, which belongs solely to the test data.
+
+# %%
+# Under Annotated Segments
+# ------------------------
 #
-# This check is the
-# :class:`PropertyLabelCorrelation <deepchecks.nlp.checks.data_integrity.property_label_correlation.PropertyLabelCorrelation>`
-# check, which will check the correlation between the properties and the labels, and report back on any properties that
-# have a high correlation with the labels.
+# Another note-worth segment is the Under Annotated Segment check, which
```

Comment:
- phrasing - did you mean "check" instead of segment?
- note-worth or note-worthy?
- add link to check plot / API reference? (on the "Under Annoted Segment" text)